ISSUE_TEMPLATE.md

```diff
@@ -2,30 +2,20 @@ Hey there and thank you for using React Native!
 
 React Native, as you've probably heard, is getting really popular and truth is we're getting a bit overwhelmed by the activity surrounding it. There are just too many issues for us to manage properly.
 
-Please read the following tips before filing an issue:
+Do the checklist before filing an issue:
 
-Is this something you can debug and fix?
--------------------------------------------------------
+- [ ] Is this something you can **debug and fix**? Send a pull request! Bug fixes and documentation fixes are welcome.
+- [ ] Have a usage question? Ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/react-native). We use StackOverflow for usage question and GitHub for bugs.
+- [ ] Have an idea for a feature? Post the feature request on [Product Pains](https://productpains.com/product/react-native/). It has a voting system to surface the important issues. GitHub issues should only be used for bugs.
 
-Can you see anything in the logs? Can you debug why this happens? **Send a pull request**! Bug fixes and documentation fixes are very welcome.
 
-Have a question or not sure you've found a bug?
------------------------------------------------------------------
-Please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/react-native) with the tag `react-native`. StackOverflow is amazing for Q&A. Because of the reputation system it is likely the community will see and answer your question there.
-
-Make sure you **include a short code snippet to demonstrate the problem**.
-
-Have an idea for a feature?
-------------------------------------
-Post the feature request on [Product Pains](https://productpains.com/product/react-native/). It has a voting system that lets us focus on the most important issues the community is experiencing. The higher number of votes the more likely it is a feature will get implemented. You could always implement it yourself, too.
-
-None of the above, want to create a GitHub issue
+None of the above, want to create a bug report
 ------------------------------------------------------------------
 
 Make sure to add **all the information needed to understand the issue** so that someone can help. If the info is missing we'll add the 'Needs more information' label and close the issue until there is enough information.
 
-- If this is a bug, provide a **minimal code snippet** that reproduces it
-- Provide **screenshots** where appropriate
-- What's the **version** of React Native you're using?
-- Does this occur on iOS, Android or both?
-- Are you using Mac, Linux or Windows?
+- [ ] Provide a **minimal code snippet** that reproduces the bug
+- [ ] Provide **screenshots** where appropriate
```

Comment:
We should provide a sample GitHub repo for people to fork and quickly hack and example.